import/popups/PopupOpener.qml

```diff
@@ -34,6 +35,13 @@ Timer {
 
     property Component _contextMenuComponent
 
+    property Notice geolocationDisabledNotice: Notice {
+        duration: 3000
+        //% "Geolocation is disabled"
+        text: qsTrId("sailfish_components_webview_popupopener-la-geolocation_disabled")
+        verticalOffset: -overlay.toolBar.toolsHeight * 4
```

Comment:
This cannot be like this as this is declared in the browser. Email or any other embedder would fail.

One idea is that sailfish-components-webview has a qml singleton for theming - WebViewTheme like component (not sure what we'd call it). Browser could then override that notificationOffset value. Default could that -Theme.itemSizeMedium.

To be noted as well that TextSelectionController has Notice component.